File: Chatroom_java/app/src/main/java/com/example/chatroom_java/data/Chats.java
package com.example.chatroom_java.data;

import android.content.res.Resources;
import com.example.chatroom_java.R;

import java.util.ArrayList;
import java.util.List;


public class Chats {
    public List<Chat> chatList(Resources resources) {
        List<Chat> chatList = new ArrayList<>();

        chatList.add(new Chat(
                1,
                "user4",
                R.drawable.user_image4,
                resources.getString(R.string.text1)
        ));

        chatList.add(new Chat(
                2,
                resources.getString(R.string.user2_name),
                R.drawable.user_image2,
                resources.getString(R.string.text2)
        ));

        return chatList;
    }
}

File: Chatroom_hackthon/app/src/main/java/com/example/chatroom_hackthon/data/Chats.kt
package com.example.chatroom_hackthon.data

import android.content.res.Resources
import com.example.chatroom_hackthon.R


fun chatList(resources: Resources): List<Chat> {
    return listOf(
        Chat(
            id = 1,
            name = "user4",
            image = R.drawable.user_image4,
            text = resources.getString(R.string.text1)
        ),
        Chat(
            id = 2,
            name = resources.getString(R.string.user2_name),
            image = R.drawable.user_image2,
            text = resources.getString(R.string.text2)
        ),
    )
}